### PR TITLE
Update nouns.json

### PR DESCRIPTION
**Pull request by encosllo**

### CODE REVIEW

**Review by Besnn — APPROVED** [review]
All good - btw have you checked the trello board I opened for this thing? because you added more or less what I dotted down yesterday evening. If not drop me an email, I'll link you to it.

**Comment by encosllo** [comment]
@Besnn didn't know about this! You can use ecosmeuv@gmail.com.

**Comment by thosgood** [comment]
I haven't had a chance to look at the Trello board yet, but I will later on this week! thanks :-)

**Comment by encosllo** [comment]
Hi @thosgood ,

Just for you to know.. there are a few translations in this update that haven't been included yet.
Another thing, how can we start adding new words and not only translations of existing ones?

**Comment by encosllo** [comment]
and one more thing ... I know some people that can help us translating from basque. 
Is it possible to add this language as well?

**Comment by thosgood** [comment]
If you'd like to add new entries then just follow the layout of existing entries! Namely

- make sure that the `root` key has entries for **all** currently existing languages, even if they're empty
- make sure that every noun has an `adjs` key, even if it's just the empty object `{}`
- the "id" for a noun or an adjective is the Adler32 hash of the atom in English, e.g. the Adler32 hash of `number` is `08f9028a` (to calculate this you can use https://hash.online-convert.com/adler32-generator for example)

basically, have a go and submit a pull request, and I'll let you know if there's anything that needs changing! 😄

**Comment by encosllo** [comment]
Thanks!!

**Comment by thosgood** [comment]
> and one more thing ... I know some people that can help us translating from basque.
> Is it possible to add this language as well?

sure! have a look at `languages.json` and add an entry for Basque there (using the [ISO code](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) `eu`, I think)
